RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -1114,7 +1089,31 @@ int freeRestsRemaining();
 boolean uneffect(effect toRemove);
 boolean useCocoon();
 
+########################################################################################################
+//Wrapper for get_campground(), primarily deals with the oven issue in Ed.
+//Also uses Garden item as identifier for the garden in addition to what get_campground() does
+int[item] auto_get_campground();					//Defined in autoscend/auto_util.ash
+
+
+//Returns how many Hero Keys and Fat Loot tokens we have.
+//effective count (with malware) vs true count.
+int towerKeyCount(boolean effective);			//Defined in autoscend/auto_util.ash
+int towerKeyCount();							//Defined in autoscend/auto_util.ash
+
+//Determines if we need ore for the trapper or not.
+boolean needOre();								//Defined in autoscend/auto_util.ash
+
+//Wrapper for my_path(), in case there are delays in Mafia translating path values
+string auto_my_path();							//Defined in autoscend/auto_util.ash
+
+//Visits gnasir, can change based on path
+void auto_visit_gnasir();
 
+//Item disambiguation functions
+boolean hasSpookyravenLibraryKey();				//Defined in autoscend/auto_util.ash
+boolean hasILoveMeVolI();						//Defined in autoscend/auto_util.ash
+boolean useILoveMeVolI();						//Defined in autoscend/auto_util.ash
```

Comment:
All of these except `auto_my_path` seem like they should be in appropriate quest files, not auto_util.ash

EDIT: You were already on top of this, I see.

Reply:
Oh, I did not think to move towerkeycount for some reason. let me move that too

Reply:
ok. moved towerkeycount as well.
that only leaves `string auto_my_path()` which I can't think of a place to move so i guess it can stay in auto_util.ash